R/utils.R

```diff
@@ -1,3 +1,7 @@
 naomi_write_csv <- function(...) {
   write.csv(..., row.names = FALSE, na = "")
 }
+
+naomi_read_csv <- function(...) {
```

Comment:
why do these have a `naomi_` prefix?

Reply:
Other parts of the package are relying on `read_csv` from `readr` package so I am trying to distinguish between those.